src/test/java/ru/ayaz/MessageProcessorTest.java

```diff
@@ -13,35 +13,36 @@ public void before() {
         processor = new MessageProcessor();
     }
 
+
     @Test
-    public void falseWhenSharpNotFirst() {
+    public void shouldNotBeTreatedAsCommandWhenCharacterComesFirst() {
         assertEquals(false, processor.isCommand("d#quit"));
     }
 
     @Test
-    public void trueWhenSharpFirstAfterSpaces() {
+    public void shouldBeTreatedAsCommandWhenSharpComesFirstAfterSpaces() {
         assertEquals(true, processor.isCommand("    #quit"));
     }
 
     @Test
-    public void shouldGetCommandWhenNormalStartSpaces() {
+    public void shouldGetCommandWhenSharpComesFirstStartSpaces() {
```

Comment:
&#x1F34E; **Issue**
I think the ending "StartSpaces" makes no sense, could you fix it ?